docs/imageomics/treeoflife10m.md

```diff
@@ -1,9 +1,34 @@
-# Dataset Card for TreeOfLife-10M
+# How to Create TreeOfLife-10M
 
+**Note:** [TreeOfLife-10M](https://huggingface.co/datasets/imageomics/TreeOfLife-10M) has the EOL images, but not iNat21 or BIOSCAN-1M due to licensing restrictions. To reconstruct the _full_ dataset, please follow the steps outlined below in Reproduce TreeOfLife-10M.
 
```

Comment:
Perhaps another note that this setup is meant to be run on a HPC system using Slurm.